_sass/_pulumi.scss

```
@@ -929,3 +929,109 @@ h4 { font-size: 22px; font-weight: 500; }
 table.clouds-table td {
     padding: 16px;
 }
+
+//
+// Styling for Python documentation
+//
+
+// Our Python documentation is generated by the Sphinx HTML generator. We use it
+// to generate unstyled HTML that we inject directly into our web page. Naturally,
+// we want to style it a bit to make it not look terrible.
+//
+// Our style selectors are split into two toplevel selectors: a div starting with "module-",
+// and a div with class "section" whose ID begins with "pulumi". These two things identify
+// the two kinds of pages that Sphinx emits: the first kind is a module page, containing
+// information on on members within a module, and the second kind is a table of contents.
+
+// This selector is for the module page.
+div[id^="module-"] {
+    // Don't show the header link - there's an anchorjs link on this header that does the
+    // job just as well.
+    h1[id$="¶"] {
```

BTW, I was running into the following error when trying to build:

```
  Conversion error: Jekyll::Converters::Scss encountered an error while converting 'css/styles.scss':
                    Invalid US-ASCII character "\xC2" on line 950
```

Looks like [escaping](https://www.w3.org/International/questions/qa-escapes#cssescapes) the character fixes the issue:

```suggestion
    h1[id$="\B6"] {
```